### DISCUSSION TITLE
Map orientation

### DISCUSSION THREAD

**Discussion by Seevetal19** [devemux86/cruiser#564] [Q&A] [answered]
Hello
I have installed Cruiser and used it for the first time, not yet bought it. I noticed something, possibly an operating error: As soon as I activate the follow mode (the circle symbol at the bottom right), the view switches to the direction of travel. I set the map orientation to "no rotation" in the settings, but that doesn't change anything. How can I change this?

A second question: When I tap on the compass symbol at the top left in Follow mode, a window appears with two options: Add Favorite and Screen Lock. I actually expect to be able to change the map orientation (as with Kurviger 1).
![image](https://github.com/devemux86/cruiser/assets/171522553/5a18b3f6-2340-4426-815e-c5502bbb773a)

**Comment by devemux86** [comment, marked as answer]
Hi @Seevetal19,

To change the map orientation, please **long press** on the compass while in navigation or follow mode.
(the changed map orientation is saved for the future)

Tap on the compass is now used for the new quick actions while traveling.

**Reply by Seevetal19** [reply]
Many thanks for the quick tip!

**Comment by devemux86** [comment]
@Seevetal19 

We generally now use long press for menu and tap for actions.
